From 02002811fc7967f351525ad1b103db07dc2b15c0 Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Wed, 28 Apr 2021 23:40:13 +0800
Subject: [PATCH 1/2] LF-3760 media: csi: fill in colorspace

filling in colorspace, this also fixed v4l2 compliance test issue.

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Reviewed by: G.n. Zhou <guoniu.zhou@nxp.com>
(cherry picked from commit 9db46794ad9925f34979936755818f08ba0c73d4)
---
 drivers/media/platform/mxc/capture/mx6s_capture.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/media/platform/mxc/capture/mx6s_capture.c b/drivers/media/platform/mxc/capture/mx6s_capture.c
index 4b61e957d19b..39bef1e160b9 100644
--- a/drivers/media/platform/mxc/capture/mx6s_capture.c
+++ b/drivers/media/platform/mxc/capture/mx6s_capture.c
@@ -1435,6 +1435,10 @@ static int mx6s_vidioc_try_fmt_vid_cap(struct file *file, void *priv,
 	pix->sizeimage = fmt->bpp * pix->height * pix->width;
 	pix->bytesperline = fmt->bpp * pix->width;
 
+	pix->colorspace = V4L2_COLORSPACE_SRGB;
+	pix->ycbcr_enc = V4L2_MAP_YCBCR_ENC_DEFAULT(pix->colorspace);
+	pix->quantization = V4L2_QUANTIZATION_FULL_RANGE;
+
 	return ret;
 }
 
-- 
2.25.1

